src/components/themed/TransactionListTop.tsx

```diff
@@ -409,7 +409,7 @@ export class TransactionListTopComponent extends React.PureComponent<Props, Stat
 
     // Fiat Balance Formatting
     const exchangeAmount = convertNativeToDenomination(exchangeDenomination.multiplier)(nativeBalance)
-    const fiatBalance = mul(exchangeAmount, exchangeRate)
+    const fiatBalance = mul(exchangeAmount, exchangeRate.toString())
```

Comment:
Should this be done as `parseFloat(exchangeAmount) * exchangeRate`? The `formatNumber` function takes `number | string`, so it's fine.